fobi_custom/override_theme/templates/override_theme/form_entry_submitted_ajax.html

```diff
@@ -0,0 +1,5 @@
+<div class="col col-lg-10 card">
+  <div class="card-body">
+    <h2 class="card-title">Survey submitted!</h2>
+  </div>
```

Comment:
It'd be nice to have two links here:

- `Back to surveys`
- `Run again`

Also, perhaps out of the scope of this PR, but I find myself wanting to preview the saved data on this view. If that's simple to do out of the box in Fobi we can add it here, otherwise I'll open up an improvement issue for it.

Reply:
It's not built into Fobi! Go ahead and open another issue—we'll just have to find a way that works for both survey admins and individuals filling out the survey for themselves.

Reply:
(Buttons added though, great idea!)